pom.xml

```diff
@@ -145,6 +145,10 @@
     <version.bundle.plugin>2.5.3</version.bundle.plugin>
     <!-- Surefire version 2.19.1 coming from parent is buggy and kie-osgi tests in droolsjbpm-integration are failing because of that. -->
     <version.surefire.plugin>2.18.1</version.surefire.plugin>
+    <!-- Jacoco plugin configurations -->
+    <jacoco.haltOnFailure>false</jacoco.haltOnFailure>
+    <jacoco.limit.minimum>0.9</jacoco.limit.minimum>
```

Comment:
Sorry for not noticing before, but this should have a more descriptive name. What about `jacoco.lineCoverageLimit`?